src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  width: 96px;
+  justify-content: space-between;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+}
+
+.stars--4 :last-child {
```

Comment:
Add solution for all modifiers `.stars--0`, `.stars--1`, `.stars--2`, `.stars--3`, `.stars--5`